discovery_engine_core/web-api/src/bin/ingestion.rs

```diff
@@ -354,8 +371,25 @@ async fn handle_rejection(err: Rejection) -> Result<impl Reply, Infallible> {
     let json = warp::reply::json(&ErrorMessage {
         code: code.as_u16(),
         message: message.into(),
+        log_id,
         errored_ids,
     });
 
-    Ok(warp::reply::with_status(json, code))
+    Ok(Box::new(warp::reply::with_status(json, code)))
+}
+
+fn setup_request_span(handler_name: &str) -> (Span, Uuid) {
+    //FIXME use v6/v7 once supported/standardized
+    let log_id = Uuid::new_v4();
+    let span = span!(
+        Level::INFO,
+        "request",
+        method = %handler_name,
+        log_id = %log_id,
+    );
+
+    //keep same wording as in `warp::trace::request()` for forward/backward compatibility
```

Comment:
:grey_question: what does this means exactly?

Reply:
A leftover from when I had it mirror warp::trace::request as much as possible.